### DISCUSSION TITLE
Single shortcut to perform both "Almost Maximize" and "Upper Center" to a window?

### DISCUSSION THREAD

**Discussion by greenerycomplexity** [rxhanson/RectanglePro-Community#509] [Ideas]
I love the way this config looks when working with multiple windows, was wondering if you can implement it as a feature to then be assigned a keyboard shortcut to. 

Right now being only 2 clicks away is still plenty fast already, but would be great if it could be done in one.

Amazing app, appreciate your hard work!

**Comment by rxhanson** [comment]
Thanks for the feedback. This is on my backlog, but in a little bit different form - allowing custom size/pos to have an offset, so that it isn't in just center or edge. I'll post back here when that one makes it in.
